local/bin/py/build/actions/integrations.py

```diff
@@ -28,7 +28,8 @@
 from actions.format_link import format_link_file
 
 try:
-    from assetlib.constants import CLASSIFIER_TAGS
+    from assetlib.classifiers import get_all_classifier_names, get_non_deprecated_classifier_names
+    CLASSIFIER_TAGS = get_all_classifier_names()
```

Comment:
Is the importError needed here? Why not instead import at the top of the file and let python fail if the assetlib isn't there?

Reply:
So the only reason I did that was just because some people may not have the library installed locally but still want the rest of the script to process.